### ISSUE TITLE
Create JRS regression test for Smart Contract service

### ISSUE THREAD

**Issue by ljianghedera** [hashgraph/hedera-services#366]
Summary
Create JRS regression test workflow for Smart Contract service, just like the one for File service. 

Creating a separate issue so people can work on it in parallel. 

Possible resolution

1. Create new test configuration for Smart Contract services and run it manually on AWS instances;
